llvm/lib/Target/RISCV/RISCVInstrInfoXCV.td

```diff
@@ -662,6 +662,7 @@ class CVLoad_ri<bits<3> funct3, string opcodestr>
 
 let Predicates = [HasVendorXCVelw, IsRV32], hasSideEffects = 0, 
```

Comment:
If this is going to put the processor to sleep until an event happens, you probably want hasSideEffects=1 so other non-event loads can't move around it.